.github/workflows/docs-build.yaml

```diff
@@ -10,7 +10,7 @@ on:
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}
-  TAG: 0.3.0
+  TAG: 0.4.0
```

Comment:
I need to update this tag so that when it's merged the build noticed that `0.4.0` does not exist and builds the `0.4.0` image that has the updated theme and so on.